main.go

```diff
@@ -80,6 +81,7 @@ func main() {
 	storeBuilder := store.NewBuilder()
 
 	ksmMetricsRegistry := prometheus.NewRegistry()
+	registerBuildInfo(ksmMetricsRegistry)
```

Comment:
@yeya24 Any reason why you didn't consider the existing collector? https://github.com/prometheus/common/blob/master/version/info.go

Reply:
Nice Kemal, agreed!

Reply:
@kakkoyun I was thinking about using that directly. But that works well with `promu` because `promu` injects all the info via build flags to that pkg. 

Right now the injected flags are added https://github.com/kubernetes/kube-state-metrics/blob/master/Makefile#L55 here, I still need to add more flags to make the `VersionCollector` usable. I feel it is not necessary.

Reply:
True. I have done exactly what you said in here https://github.com/kakkoyun/observable-remote-write/blob/master/Makefile 

Hard to judge just which way is simpler. I guess it's a matter of preference. My preference would be obviously reusing the package from the `prometheus/common`.

@lilic What do you think? Which code looks simpler to you?

Reply:
Agreed, reusing Prometheus/common would be my preference here as well. 👍

Reply:
Got it! I will use that package. Thanks @kakkoyun for the example!